.docker/run-gui-moveit-docker

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# This script is used to create and run a persistent docker container with settings for GUI. See the instructions online.
+# Delete/restart the container by running `docker stop local_moveit_container && docker rm local_moveit_container`
+
+#!/bin/bash
+
+XAUTH=/tmp/.docker.xauth
+if [ ! -f $XAUTH ]
+then
+    xauth_list=$(xauth nlist :0 | sed -e 's/^..../ffff/')
+    if [ ! -z "$xauth_list" ]
+    then
+        echo $xauth_list | xauth -f $XAUTH nmerge -
+    else
+        touch $XAUTH
+    fi
+    chmod a+r $XAUTH
+fi
+
+if [ ! "$(docker ps -aq -f name=local_moveit_container)" ]
+then
+    echo "Docker container does not exist yet. Running from scratch"
+    docker run -it \
+    --env="DISPLAY=$DISPLAY" \
+    --env="QT_X11_NO_MITSHM=1" \
+    --volume="/tmp/.X11-unix:/tmp/.X11-unix:rw" \
+    -env="XAUTHORITY=$XAUTH" \
+    --volume="$XAUTH:$XAUTH" \
+    --volume="/home/parallels/moveit-dev/catkin_ws:/root/linked_ws_moveit" \
+    --runtime=runc \
+    --name local_moveit_container \
+#    -v /home/your_username/moveit_ms:/root/ws_moveit_linked \
+    -w /root/linked_ws_moveit \
+    my_moveit_image
```

Comment:
The only way I could get this to run was by changing `my_moveit_image` to `local_moveit_image`. Should that be necessary? I kept getting `docker: Error response from daemon: pull access denied for my_moveit_image, repository does not exist or may require 'docker login'.` before switching.

Reply:
My bad, I didn't coordinate that change between the two PRs. `local_moveit_image` should be correct.